### PR TITLE
Write documentation for sugar3.power

### PR DESCRIPTION
**Pull request by quozl**
- get it done before someone else tries it,
- tested using `make-doc.sh`,
- following [documentation style](https://wiki.sugarlabs.org/go/Features/Sugar3_Docs/How_To_Write); and,
- with method references [Cross-referencing Python objects](http://www.sphinx-doc.org/en/1.5.1/domains.html#cross-referencing-python-objects).

### CODE REVIEW

File: src/sugar3/power.py [hunk 1]
```diff
@@ -15,6 +15,17 @@
...
 
+"""The power module provides an interface to *powerd*, a daemon that
+manages the aggressive suspend and wakeup policies for early OLPC
+laptops.
+
+The module does nothing if *powerd* is not present.  *powerd* is not
+required on laptops other than OLPC XO-1, XO-1.5, XO-1.75 and XO-4.
+Distributions of Sugar on other hardware need not include the *powerd*
+package.
+
```
Comment (i5o) [inline src/sugar3/power.py]: Can we remove this newline?

File: src/sugar3/power.py [hunk 2]
```diff
@@ -15,6 +15,17 @@
 # Free Software Foundation, Inc., 59 Temple Place - Suite 330,
 # Boston, MA 02111-1307, USA.
 
+"""The power module provides an interface to *powerd*, a daemon that
```
Comment (i5o) [inline src/sugar3/power.py]: As I've seen, most of new documentation is written after the 
`"""`
can you change it?

File: src/sugar3/power.py [hunk 3]
```diff
@@ -24,16 +35,48 @@
...
+    continue to dim and blank the display.
+
+    Both the :py:meth:`inhibit_suspend` and :py:meth:`restore_suspend`
+    methods are reference counted; automatic idle suspend is not
+    restored until the same number of calls to restore are made.
+
+    *powerd* is resilient against failure to restore automatic idle
+    suspend; it verifies an inhibit request and deletes it if the
+    requesting process has terminated.
+
```
Comment (i5o) [inline src/sugar3/power.py]: same with this newline and the others

**Comment by quozl** [comment]
Sure, easily done, 3d4bf93.  The format came from emacs fill paragraph feature in python mode and I didn't have any reason to object to it.  But "most new documentation" is a good reason.

**Comment by quozl** [comment]
Rebased to master.

File: src/sugar3/power.py [hunk 4]
```diff
@@ -47,9 +91,18 @@ def __del__(self):
...
+        toolkit?  Yes.  For future use by a toolkit with more
+        resilient collaboration.
+
+        Returns:
+            True
+        """
         return True
 
     def inhibit_suspend(self):
+        """Inhibit automatic idle suspend until restored."""
```
Comment (i5o) [inline src/sugar3/power.py]: We also leave a newline here; as seen in
https://github.com/sugarlabs/sugar-toolkit-gtk3/blob/master/src/sugar3/graphics/xocolor.py#L279
https://github.com/sugarlabs/sugar-toolkit-gtk3/blob/master/src/sugar3/graphics/toolbutton.py#L206
etc

**Comment by quozl** [comment]
Fixed in 3659570.  A pity, as vertical space is always the limiting factor when a new developer is trying to comprehend code they've never seen before.

**Comment by i5o** [comment]
Merged 
https://github.com/sugarlabs/sugar-toolkit-gtk3/commit/99130b02639600f2645667eb514ef0eea4df0fa5

**Pull request closed without merging**